poo/abstraction/en/Abstraction.md

```diff
@@ -0,0 +1,117 @@
+# Abstraction
+
+The **abstraction** idea, is chopper a class in parts, to be able to share (via a parameter passed in method for example) **an only** of this part rather the object “in full”.
+To understand the utility of this principle and how apply, take for example a class Human, since this is a thing that we know better.
+
+```java
+
+class Human {
+
+    int air;
+    int satiety;
+    float energy;
+
+
+    void breathe() {
+        air += 10;
+    }
+
+    void eat() {
+        satiety += 5;
+    }
+
+    void move() {
+        System.out.println("Un humain souhaite se déplacer !");
+    }
+
+    void consumeEnergy(float value) {
+        this.energie -= value;
+    }
+}
+```
+
+Now, imagine that we have a class Hiking which allows organise a walk enter Humans. A thing of style:

```

Comment:
I think this part is incorrect:
> a class Hiking which allows organise a walk enter Humans

I would have translated that as ```a class `Hiking` that allows us to organise/organize a walk[...]```
Also the text is not formatted as the original. `Hiking` is in code block and Human is not written as a class in the French version.
I need help to translate `une marche entre humains` because I'm really not certain with the current translation `a walk enter Humans`.

Reply:
The text formatting has been checked and corrected
`a class Hiking which allows organise a walk` has been replace to `a class Hiking that allows us to organise a walk`

Reply:
Thanks